src/cloud_common/protocols/unit/desired_status.py

```diff
@@ -0,0 +1,431 @@
+#
+#  Copyright (c) 2018-2024 Renesas Inc.
+#  Copyright (c) 2018-2024 EPAM Systems Inc.
+#
+import base64
+from datetime import datetime, time
+from typing import Annotated, Literal
+
+from pydantic import Base64Bytes, BaseModel, Field, field_serializer
+
+from cloud_common.protocols.unit.types import (
```

Comment:
We discussed that system alerts should be configured by cloud for each node. I don't see this in the protocol.

Another missing thing it is default request resource ration as I remember.